model/src/cfd.rs

```diff
@@ -296,6 +298,35 @@ impl Order {
             self.opening_fee,
         )
     }
+
+    const OUTDATED_AFTER_MINS: i64 = 10;
+
+    /// Defines when we consider the order to be sane within the system boundaries
+    ///
+    /// This is a safety net for the taker to prevent the taker from taking an outdated order.
+    pub fn is_safe_to_take(&self, now: OffsetDateTime) -> bool {
```

Comment:
This seems to be what you're trying to convey and also you can use it without negating it.

```suggestion
    pub fn is_outdated(&self, now: OffsetDateTime) -> bool {
```

Reply:
That's what the error message returned says, but internally the check touches more than just being outdated.
Initially I had `is_outdated` but I went away from it because I felt it was not precise enough. 
I would keep as is for now.

Reply:
Fair enough. Both conditions are about time, but the one related to the oracle settlement time is not necessarily about it being outdated.